test/unit/services/themes/middleware_spec.js

```diff
@@ -5,6 +5,7 @@ const themes = require('../../../../core/frontend/services/themes');
 // is only exposed via themes.getActive()
 const activeTheme = require('../../../../core/frontend/services/themes/active');
 const settingsCache = require('../../../../core/server/services/settings/cache');
+const stats = require('../../../../core/server/services/stats');
```

Comment:
Would need some more unit tests covering new helper itself, similarly to [these tests](https://github.com/TryGhost/Ghost/blob/b34f2e86fb9a785e7483f6ffc4a1b0269f9f6709/test/unit/helpers/pagination_spec.js).

Reply:
It seems that the wrong example is given. `{{@stats}}` is an global object, not helper functions. The pagination example and other tests inside that folder are about testing helper functions in `core/frontend/helpers`, not the global objects passed in `frontend/services/themes/middleware.js`. 

It does have the cases that use global objects like `@site` in `test/unit/helpers/concat_spec.js`, but they're testing `has` and `@site` is set up as `defaultGlobals` in `before`. The values are loaded from the different route. 

I think the best alternative is to use `test-theme` in the fixture or create a new theme to use `{{@stats}}` helper. Then, write an acceptance test that activates that theme and load the page to check if `{{@stats}}` are loaded correctly. 

But it sounds a bit big. Do you want me to do that?

Reply:
I slept on it for days and decided that it's the only solution we have for this case and the case for updating stats when something changed. 

I'll go down this road tomorrow.